source/storage.rst

```diff
@@ -0,0 +1,159 @@
+.. toctree::
+
+Storage on EGI Cloud
+====================
+
+This page aims to give a brief description of the storage features provided by
+the EGI Cloud and a basic tutorial on how to use and integrate them into your
+application.
+
+If you are in need of more storage than the one provided within image disk of
+your VMs, or need to persist data independently from the VMs storage, you will
+need to use one of the solutions available on the EGI service catalogue.
+
+There are two kind of storage services, Block and Object. Both of them have
+their own set of advantages and disadvantages
+
+**Block storage** provides additional storage blocks which can be attached to a
+virtual machine. A storage block is a virtual disk of a given size, which can 
+be exposed as a virtual device in the VM. You can think of this can of devices
+as a USB stick that can be plugged into the VMs and can be used as a regular
+drive. You can format it with any file system you want and mount it in your VM.
+Block devices are persistent, thus they keep all the data after VM shutdown and
+need to be explicitly destroyed when data is not needed anymore. Block storage
+disks can be accessed only from within a VM, and only from VMs running at the
+same provider where the block storage is located. Also, they can be accessed by
+only one VM at the same time. As part of the IaaS service, block storage is
+managed via OpenStack/OCCI APIs. There is a limit on the number of block
+storage devices you can attach on a VM and there is a limit to the maximum
+size of such virtual disks. These values will depend on the particular provider
+and your SLA. Moreover, the disk space is accounted for the entire block
+storage device, regardless how much of it is actually used. 
+
+**Object storage** is a standalone service that stores data as set of individual
+objects, which can have different types (e.g. files, images, documents) and
```

Comment:
Do objects in swift really have a type? Or is the intent to say that objects can be of any/various file types?

Reply:
I don't see much things here: https://docs.openstack.org/api-ref/object-store/index.html?expanded=get-object-content-and-metadata-detail#objects

Reply:
the intent is to say objects can be of whatever type, I'm rewording